pkgs/applications/networking/browsers/misc/widevine-cdm.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchzip
+}:
+
+stdenv.mkDerivation rec {
+  pname = "widevine-cdm";
+  version = "4.10.2449.0";
+
+  src = fetchzip {
+    url = "https://dl.google.com/widevine-cdm/${version}-linux-x64.zip";
+    sha256 = "sha256-f2kAkP+s3fB+krEZsiujEoI4oznkzSyaIB/CRJZWlXE=";
+    stripRoot = false;
+  };
+
+  installPhase = ''
+    install -vD libwidevinecdm.so $out/libwidevinecdm.so
```

Comment:
I made my widevine cdm package years ago, but at the time [patching](https://github.com/nrdxp/nrdos/blob/master/src/display/packages/widevine.nix#L26-L28) was required or it wouldn't work correctly.

I am assuming you have tested this and it works without this rpath patch, but I just wanted to ask you explicitly to make sure.

Reply:
Thank you for pointing that out. Yes I did test it and both Spotify and Prime Video worked on an Archlinux based install (without the AUR package `qt5-webengine-widevine` obviously) and a NixOS install, so I am pretty confident that it is not required.